__tests__/compilers/compatability.test.tsx

```diff
@@ -409,4 +409,59 @@ ${JSON.stringify(
       "
     `);
   });
+
+  it('compiles tables with inline code with newlines', () => {
+    const md = `
+[block:parameters]
+${JSON.stringify(
+  {
+    data: {
+      'h-0': 'Field',
+      'h-1': 'Description',
+      '0-0': 'orderby',
+      '0-1': '`{\n "field": "ID",\n "type": "ASC"\n }`',
```

Comment:
What would happen if they'd wrapped this in a code block instead of an inline? Would that "just work"?

Reply:
I _think_ so. RDMD should compile it back to magic block.